flink/src/test/java/org/apache/iceberg/flink/TestHelpers.java

```diff
@@ -49,6 +61,49 @@
   private TestHelpers() {
   }
 
+  public static RowData copyRowData(RowData from, RowType rowType) {
```

Comment:
@JingsongLi , I can not use `RowDataSerializer` directly since the returned RowData may contain metadata column after merging with position deletes. So I created this function to do the copy job.